packages/server/src/tests/epics/choiceCards.ts

```diff
@@ -0,0 +1,166 @@
+import { ChoiceCardAmounts, EpicTest } from '@sdc/shared/src/types/epic';
+import { epic, epicWithChoiceCards } from '@sdc/shared/src/config/modules';
+import { UK_DATA, EU_DATA, ROW_DATA, US_DATA, CTAS } from './choiceCardsTestData';
+import { CountryGroupId } from '@sdc/shared/dist/lib';
+import { ArticlesViewedSettings } from '@sdc/shared/types';
+
+const testName = '2021-08-25-EpicChoiceCardsTest';
+
+export enum EpicChoiceCardsTestVariants {
+    control = 'control',
+    variant1 = 'variant1',
+    variant2 = 'variant2',
+}
+
+const buildEpicChoiceCardsTest = (
+    locations: CountryGroupId[],
+    suffix: string,
+    paragraphs: string[],
+    highlightedText: string,
+    choiceCardAmounts: ChoiceCardAmounts,
+    articlesViewedSettings?: ArticlesViewedSettings,
+): EpicTest => ({
+    name: `${testName}__${suffix}`,
+    campaignId: testName,
+    isOn: true,
+    locations,
+    audience: 1,
+    tagIds: [],
+    sections: [],
+    excludedTagIds: [],
+    excludedSections: [],
+    alwaysAsk: false,
+    maxViews: {
+        maxViewsCount: 4,
+        maxViewsDays: 30,
+        minDaysBetweenViews: 0,
+    },
+    userCohort: 'AllNonSupporters',
+    isLiveBlog: false,
+    hasCountryName: true,
+    variants: [
+        {
+            name: EpicChoiceCardsTestVariants.control,
+            modulePathBuilder: epic.endpointPathBuilder,
+            paragraphs: paragraphs,
+            highlightedText: highlightedText,
+            cta: CTAS.control,
+            separateArticleCount: { type: 'above' },
```

Comment:
`separateArticleCount` should not be set for the 'top readers' tests

Reply:
👍🏼